api_formatter/templates/rest_framework/pagination/numbers.html

```diff
@@ -1,46 +1,46 @@
-<ul class="pagination">
```

Comment:
I'm looking at this fresh, and I think we need to update this component in the Style Library. We don't need any of these `tabindex` values, because they can all be focusable. I think the `...` between page numbers should have:
1. The `disabled` class (and `aria-disabled=true` added to the link)
2. An aria-label that says "truncated pages"